ckanext/agsview/public/js/ags_fs_view.js

```diff
@@ -86,6 +86,11 @@ ckan.module('ags_fs_view', function (jQuery, _) {
     getMetaData: function () {
       var self = this;
       this.layer.metadata(function(error, metadata){
+          Object.keys(metadata.fields).forEach(function(key) {
+            if (metadata.fields[key].type == 'esriFieldTypeDate') {
+              Date_names.push(metadata.fields[key].name)
```

Comment:
Where is `Date_names` variable coming from?

Reply:
This is a new variable defined to record the keys of data whose type is 'esriFieldTypeDate'. Data with this type is epoch time and need to be converted to datetime